### PR TITLE
chore(scripts): ensure changes in language folder before releasing

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-1646

### Changes included:

because we allow global scopes bumping (e.g. `clients` or `specs`), we don't check if there was actual changes in a client before bumping its version.

this PR aims at ensuring there's a change in a client before considering it for a release.

### CODE REVIEW

**Comment by shortcuts** [comment]
it works https://github.com/algolia/api-clients-automation/pull/2261

File: scripts/release/__tests__/createReleasePR.test.ts [hunk 1]
```diff
@@ -37,6 +37,22 @@ vi.mock('../../common.js', async () => {
   }
 })
 
+vi.mock('../../ci/utils.js', async () => {
+  const mod = await vi.importActual<typeof import('../../ci/utils.js')>('../../ci/utils.js')
+  return {
+    ...mod,
+    getNbGitDiff: vi.fn().mockReturnValue(1)
```
Comment (millotp) [inline scripts/release/__tests__/createReleasePR.test.ts:44]: this should be `mockResolvedValue`, same below

**Review by millotp — COMMENTED** [review]
looks great ! small nitpick

File: scripts/release/__tests__/createReleasePR.test.ts (hunk 1 above)
Comment (shortcuts) [reply scripts/release/__tests__/createReleasePR.test.ts]: good catch

**Review by millotp — APPROVED** [review]

**Review by aallam — APPROVED** [review]
Nice! GJ!

**Review by millotp — APPROVED** [review]

**Pull request merged**